documentation/modules/security/proc-configuring-external-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='configuring-external-clients-to-trust-cluster-ca-{context}']
+= Configuring external clients to trust the cluster CA
+
+This procedure describes how to configure a Kafka client that resides outside the Kubernetes cluster – connecting to the `external` listener on port 9094 – to trust the cluster CA certificate.
+Follow this procedure when setting up the client and during the renewal period, when the old clients CA certificate is replaced.
+
+Follow the steps to configure trust certificates signed by the cluster CA for Java-based Kafka Producer, Consumer, and Streams APIs.
+
+Choose the steps according to the certificate format for the cluster CA: PKCS#12 (`.p12`) or PEM (`.crt`).
+
+IMPORTANT: The `_<cluster-name>_-cluster-ca-cert` `Secret` will contain more than one CA certificate during CA certificate renewal.
+Clients must add _all_ of them to their truststores.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A `Kafka` resource within the Kubernetes cluster.
+* A Kafka client application outside the Kubernetes cluster which will connect using TLS and needs to trust the cluster CA certificate.
+
+.Using PKCS #12 format (.p12)
+
+. Extract the cluster CA certificate and password from the generated `_<cluster-name>_-cluster-ca-cert` Secret.
++
```

Comment:
What do you think about adding explanatory comments to these commands?

Reply:
If you think it's adds clarity. So what I've done, so I don't repeat the same thing in both mini procedures, is I've added this to the intro to the procedure:

> The steps describe how to:
> 
> - Obtain the certificate that verifies the identity of the Kafka cluster
> - Configure the client credentials to use the extracted certificate